app/models/user.rb

```diff
@@ -0,0 +1,5 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: :author_id
+  has_many :likes, foreign_key: :user_id
+  has_many :comments, foreign_key: :user_id
+end
```

Comment:
- [ ] Hey @IsmailChangezi great job implementing some suggestions made by the previous reviewer. I'm sure you missed this. Could you implement a method for the User model that returns (3) most recent posts?